README.md

```diff
@@ -1,24 +1,78 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Car Booking APi
 
-Things you may want to cover:
+> Car Booking API is Rais API based that allows registered users to book cars. only user of the type admin is allowed to add vehicles in  the system and other users can book them them and see their reservations
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails & postgreSQL.
+- JWT.
+- Linters Rubocop
+- Git & GitHub.
 
-* Configuration
+## Getting Started
 
-* Database creation
+To get a local copy up and running follow these simple example steps.
 
-* Database initialization
+### Prerequisites
 
-* How to run the test suite
+    Git
+    PostgreSQL
+    Ruby on Rails
+    
+    
+    
 
-* Services (job queues, cache servers, search engines, etc.)
+### Setup
 
-* Deployment instructions
+    1. git clone https://github.com/JoffreyNK/final-capstone
+    2. cd final-capstone
+    3. bundle install
+    4. rails db:create
+    5. rails db:migrate
 
-* ...
+### Usage
+
+    1. rails server
+    2. xdg-open http://localhost:3000/api/v1
```

Comment:
- [ ] Your instructions point in the end to a path that has no route:

![image](https://user-images.githubusercontent.com/57421823/201430267-ce9553ed-61c1-4154-a62d-6e761907dc37.png)

Kindly remove that direction, or add a route to it.   That way, your users won't be lured to an error page. 👍